.github/workflows/make-check.yml

```diff
@@ -26,3 +26,6 @@ jobs:
           go-version: ${{ matrix.go }}
       - name: Build
         run: make check
+      - if: runner.os == 'Linux'
+        name: Go version check
+        run: make goversioncheck
```

Comment:
Not sure yet how this will interact with https://github.com/crc-org/crc/pull/4132

Reply:
Didn't get you, it is different target the vendor verification right?

Reply:
#4132 had a change changing the go version to 1.21 in the go.mod file, but not touching the rest of the files. This would temporarily break `make go versioncheck` if this went in this way (it won't).